src/databricks/labs/remorph/reconcile/execute.py

```diff
@@ -88,12 +95,16 @@ def recon(
         except DataSourceRuntimeException as e:
             schema_reconcile_output = SchemaReconcileOutput(exception=str(e))
         else:
-            if report_type in {"schema", "all"}:
+            if report_type.lower() not in {"all", "data", "row", "schema"}:
```

Comment:
move this to a function in `execute.py` so that we can use this in multiple place, make it generic.

``` 
def validate_input(input: str, list_of_value: set, msg: str):
            if input not in list_of_value:
                 raise InvalidReport ....

Reply:
Done